docs/user/ha-enabled-sample-broker.yaml

```diff
@@ -0,0 +1,40 @@
+broker:
+ dataSource:
+  url: jdbc:derby:database
+  user: root
+  password: root
+
+transport.amqp:
+ channelFlow:
+  lowLimit: 100
+  highLimit: 1000
+
+ plain:
+  hostName: localhost
+  port: 5672
+
+ ssl:
+  enabled: true
+  protocol: TLS
+  hostName: localhost
+  port: 8672
+
+  keyStore:
+   type: JKS
+   location: resources/security/keystore.jks
+   password: wso2carbon
+   certType: SunX509
+
+  trustStore:
+   type: JKS
+   location: resources/security/client-truststore.jks
+   password: wso2carbon
+   certType: SunX509
+
+failover:
+ enabled: true
+ strategy: org.wso2.broker.coordination.rdbms.RdbmsHaStrategy
+
+rdbmsCoordinationConfig:
```

Comment:
Shall we change this to strategy and strategy option similar to how authentication module is done?